src/sql/src/statements/create.rs

```diff
@@ -34,6 +71,42 @@ pub struct CreateTable {
     pub partitions: Option<Partitions>,
 }
 
+impl CreateTable {
+    #[inline]
+    fn format_constraints(&self) -> String {
```

Comment:
This method might be too complicate
```suggestion
    fn format_constraints(&self) -> String {
```